distribution/docs/src/main/resources/content/_managing/attributes-added-on-query.adoc

```diff
@@ -7,4 +7,4 @@
 
 == {title}
 
-Metacards resulting from a query will undergo <<attributes_added_by_attribute_injection,Attribute Injection>>, then have their <<_attributes_added_by_attribute_overrides_query,Attributes Overridden>>.
+Metacards resulting from a query will undergo <<{managing-prefix}attributes_added_by_attribute_injection,Attribute Injection>>, then have their <<_attributes_added_by_attribute_overrides_query,Attributes Overridden>>.
```

Comment:
❓ Does the second link in this line need a prefix?

Reply:
fixed